e2e/webpack/src/webpack.legacy.test.ts

```diff
@@ -113,17 +114,17 @@ describe('Webpack Plugin (legacy)', () => {
     updateFile(
       `${appName}/src/main.ts`,
       `
-      document.querySelector('proj-root').innerHTML = '<h1>Welcome</h1>';
+      document.querySelector('proj-root')!.innerHTML = '<h1>Welcome</h1>';
```

Comment:
Why is this `!` necessary?

Reply:
TS type checking was finding that querySelector could possibly be null. Which is valid